src/astro/sql/operators/sql_dataframe.py

```diff
@@ -61,7 +61,7 @@ def __init__(
             **kwargs,
         )
 
-    def handle_op_args(self):
+    def load_op_arg_table_into_dataframe(self):
```

Comment:
Why don't we replace `_get_dataframe` by the `database.export_table_to_pandas_dataframe` method?